clang/test/CIR/CodeGen/switch-unreachable-after-break.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %clang_cc1 -std=c++17 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s
+// XFAIL: *
```

Comment:
These tests are failing because the insertion point is restored by an InsertionGuard in `buildSwitchBody()`. I suggest revising that part after #528 is merged to avoid conflict.

https://github.com/llvm/clangir/blob/3da10fafac66ff125fb59c602e41ad4b4f5cb382/clang/lib/CIR/CodeGen/CIRGenStmt.cpp#L972-L978